src/types/proposals.ts

```diff
@@ -796,3 +799,18 @@ export type PriorityProposal = Pick<
   linked_proposals_data?: LinkedProposal[]
   unpublished_bids_data?: UnpublishedBidInfo[]
 }
+
+export type PersonnelAttributes = TeamMember & {
+  id: string
+  project_id: string
+  status: PersonnelStatus
+  updated_by?: string
+  updated_at?: Date
+  created_at: Date
+}
+
+export enum PersonnelStatus {
+  Deleted = 'deleted',
+  Unassigned = 'unassigned',
```

Comment:
It seems like we have an extra functionality here what we aren't using or we didn't talk with the team on how to use it 🤔